examples_test.go

```diff
@@ -2,21 +2,19 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-package goftp_test
+package goftp
```

Comment:
The examples are in a different package so the godocs look correct. Why did you need to change it?

Reply:
I changed it so forking the package to another package name wouldn’t require changing the absolute import from `github.com/secsy/goftp` to `github.com/<user>/goftp`.

Is this in a separate package so the examples include the `goftp.` prefix?

Reply:
Done (a880151)